### ISSUE TITLE
Request - Add A Pre-trained model

### ISSUE THREAD

**Issue by ofirkris** [Yaoyi-Li/GCA-Matting#1]
Hi, thanks for sharing this great research.
Can you kindly share a pre-trained model?

**Comment by Yaoyi-Li** [comment]
Please have a wait. I'm writing the README.md

**Comment by tianmingdu** [comment]
Is it possible to release the paper 'GCA Matting' online?

**Comment by Yaoyi-Li** [comment]
> Is it possible to release the paper 'GCA Matting' online?

I released this paper on arxiv https://arxiv.org/pdf/2001.04069.pdf and updated the README. 😃

**Comment by ofirkris** [comment]
Thanks for sharing your work.
I'm researching ways to implement image matting on scenarios which require an extremely lightweight model.

Do you think it's possible to achieve a notable reduction in model parameters + model size  (current is 101.3MB) while keeping accuracy at high level?

**Comment by Yaoyi-Li** [comment]
I think it depends on how you define the high level accuracy. I also have tried some extremely lightweight models on mobile devices like https://arxiv.org/pdf/1905.06747.pdf. But in that case, the task is more difficult than image matting with trimap. The predictions of that model are not very stable and highly depends on the quality of input images.
IndexNet shows that mobileNet works fine in image matting. So I think it is possible to have a extremely lightweight matting model and we are also working on this.

**Comment by ofirkris** [comment]
Thanks @Yaoyi-Li  - indeed - they also have a web demo here - http://matting.zsyhh.com:4800/
End result with GCA matting is superior also due to the manual trimap needed.
